Brokerages/Oanda/OandaRestApiV20.cs

```diff
@@ -349,16 +351,14 @@ private void OnTransactionDataReceived(string json)
                     var transaction = obj.ToObject<OrderFillTransaction>();
 
                     Order order;
-                    lock (_locker)
+                    lock (Locker)
                     {
                         order = OrderProvider.GetOrderByBrokerageId(transaction.OrderID);
                     }
                     if (order != null)
```

Comment:
Should we move this `if` block inside the `lock` block as well?

Reply:
I wasn't 100% sure, because the methods that call the `api` won't free the lock until they are finished and the callback will be triggered _after_ the call to the `api`, so the thread processing the async response will wait at line354.
_But_ if a response for a previous operation comes in and we call another operation on the same order then both threads will be working on the same order 🙉.
Summary: I'll expand the `lock` block 👍